src/main/java/de/retest/web/DefaultValuesProvider.java

```diff
@@ -0,0 +1,82 @@
+package de.retest.web;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import de.retest.ui.DefaultValueFinder;
+import de.retest.ui.descriptors.IdentifyingAttributes;
+
+public class DefaultValuesProvider implements DefaultValueFinder {
+
+	public static final String DEFAULTS_FILE_PATH = "/defaults.yaml";
+
+	private final Map<String, Map<String, String>> defaultValues;
+
+	public DefaultValuesProvider() {
+		try ( final InputStream url = getClass().getResourceAsStream( DEFAULTS_FILE_PATH ) ) {
+			defaultValues = readAttributesConfigFromFile( url );
+		} catch ( final Exception e ) {
+			// TODO Own exception
+			throw new AttributesConfigLoadException( DEFAULTS_FILE_PATH, e );
+		}
+	}
+
+	private Map<String, Map<String, String>> readAttributesConfigFromFile( final InputStream in ) throws IOException {
+		final Map<String, Map<String, String>> defaultValues = new HashMap<>();
+		final ObjectMapper mapper = new ObjectMapper( new YAMLFactory() );
+		final JsonNode jsonNode = mapper.readTree( in );
```

Comment:
Why reinvent the weel? Jackson should be powerful enough to parse simple maps.

Reply:
Not if the names are unknown. If you can do better, you are welcome to improve the code. But please ... in a new branch ;-)